.github/workflows/release.yml

```diff
@@ -50,13 +50,38 @@ jobs:
       - name: Go Mod Tidy
         run: go mod tidy
 
-      - name: Go Build
+      - name: Compile for Linux amd64
         env:
           VERSION: ${{ needs.test.outputs.version }}
           CGO_ENABLED: 0
           GOOS: linux
           GOARCH: amd64
-        run: go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=$VERSION" -a -installsuffix cgo -o bin/2ms main.go
+        run: |
+          go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=$VERSION" -a -installsuffix cgo -o dist/2ms main.go
+          zip -j dist/linux-amd64-$VERSION.zip dist/2ms
+          rm dist/2ms
+
+      - name: Compile for MacOS amd64
+        env:
+          VERSION: ${{ needs.test.outputs.version }}
+          CGO_ENABLED: 0
+          GOOS: darwin
+          GOARCH: amd64
+        run: |
+          go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=$VERSION" -a -installsuffix cgo -o dist/2ms main.go
+          zip -j dist/macos-amd64-$VERSION.zip dist/2ms
+          rm dist/2ms
+
+      - name: Compile for Windows amd64
+        env:
+          VERSION: ${{ needs.test.outputs.version }}
+          CGO_ENABLED: 0
+          GOOS: windows
+          GOARCH: amd64
+        run: |
+          go build -ldflags "-s -w -X github.com/checkmarx/2ms/cmd.Version=$VERSION" -a -installsuffix cgo -o dist/2ms.exe main.go
+          zip -j dist/windows-amd64-$VERSION.zip dist/2ms.exe
+          rm dist/2ms.exe
```

Comment:
Use [`matrix`](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)